test/fixtures/db.js

```diff
@@ -0,0 +1,31 @@
+var fixtures = require('./db_test_data.js');
+var config = require('../../lib/config');
+var rethinkdbdash = require('rethinkdbdash');
+var _ = require('lodash');
+var r;
+
+var table = config.rethinkdb.table;
+
+var api = module.exports = {};
+
+api.start = function () {
```

Comment:
Re: Not being able to hook this up the the async.parallel.  If you notice in our other `start` and `stop` API calls, they accept a `callback` parameter.  This callback should be called when `start`/`stop` are completed (or have resulted in an error.